backend/src/main/java/com/redhat/cloud/notifications/routers/EndpointResource.java

```diff
@@ -510,6 +518,7 @@ public Response getDetailedEndpointHistory(@Context SecurityContext sec, @PathPa
     @PUT
     @Path("/email/subscription/{bundleName}/{applicationName}/{type}")
     @Produces(APPLICATION_JSON)
+    @Operation(summary = "Change the notification type", description = "Changes the subscription notification email type. Use this endpoint to subscribe to email notifications from a specific application.")
```

Comment:
This is inaccurate as this operation only `creates` something in the database and cannot `change/update` anything.
```suggestion
    @Operation(summary = "Subscribe to email notifications", description = "Use this endpoint to subscribe to email notifications from a specific application.")
```